library/core/src/mem/maybe_uninit.rs

```diff
@@ -1284,3 +1278,23 @@ impl<T> MaybeUninit<T> {
         }
     }
 }
+
+#[stable(feature = "foo", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
Probably using `foo` here is what broke the other test.  Please use a proper feature name.

Reply:
Doh, totally spaced on that, thanks.